### DISCUSSION TITLE
Elegoo Neptune 3 Display on another Mainboard or Klipper

### DISCUSSION THREAD

**Discussion by DerEriapp** [NARUTOfzr/Neptune_3#63] [Q&A]
Is it possible to get the LCD/display to work on another mainboard or comunicate with Klipper?

I ask because I'm building a MKS SKIPR in my Elego Neptune 3 and want to use the original display because it's nice.

I am running Klipper on my MKS SKIPR, so maybe I can connect the screen to it as Klipper screen?

I need more Information about the screen.

**Comment by bmorcelli** [comment]
Neptune 3 uses a DWIN display, that could be modded with the DGUS software, but we don't have access to the source files to work on...

Klipperscreen, as software, wont work (i believe) in this display, because it needs a dedicated firmware to do so...

 But.. there is a great soul in the klipper community who made a DWIN firmware to work with klipper.. i hadn't test it yet.. but you can find it in https://github.com/Desuuuu/DGUS-reloaded-Klipper

If you test it.. tell us if it works .

**Reply by mlee12382** [reply]
>  But.. there is a great soul in the klipper community who made a DWIN firmware to work with klipper.. i hadn't test it yet.. but you can find it in https://github.com/Desuuuu/DGUS-reloaded-Klipper


Doesn't work with the chip on the N3 screen, it will basically brick it.

**Comment by mlee12382** [comment]
No, the dgus screens aren't supported by klipper. There is one klipper branch that kind of works with a different dgus screen but it's outdated and very limited and it doesn't work with the specific chip that the N3 screen uses.
Your best option is to use KlipperScreen which connects to the host usually via hdmi or the ribbon cable but you can also set up an old android phone or tablet to run it.
If you're wanting something basic you can get a mini12864 lcd and use this https://a.co/d/0qzYnuM
